src/QueryEditor.tsx

```diff
@@ -23,6 +23,11 @@ interface LastQuery {
   format: string;
 }
 
+const blankVar: VariableQuery = {
+  query: '',
+  asJson: false,
```

Comment:
Good idea, I like that because maybe people will want csv or who knows what in the future. I had to add an extra function to deal with the boolean nature of the switch but that's not going to break anything in the future if it changes!